services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -879,6 +879,70 @@ class FlowCrudSpec extends HealthCheckSpecification {
         database.resetCosts()
     }
 
+    @Tags(LOW_PRIORITY)
+    def "System doesn't create flow when reverse path has differ bandwidth than forward path on the second link"() {
+        given: "Two active not neighboring switches"
+        def switchPair = topologyHelper.getAllNotNeighboringSwitchPairs().find {
+            it.paths.find { it.unique { it.switchId }.size() >= 4 }
+        } ?: assumeTrue("No suiting switches found", false)
+
+        and: "Select path for further manipulation with it"
+        def selectedPath = switchPair.paths.max { it.size() }
+        def altPaths = switchPair.paths.findAll { it != selectedPath }
+
+        and: "Make all alternative paths unavailable (bring ports down on the src/intermediate switches)"
+        List<PathNode> broughtDownPortsSrcSwitch = []
+        altPaths.findAll { it.first().portNo != selectedPath.first().portNo }.unique { it.first() }.each { path ->
+            def src = path.first()
+            broughtDownPortsSrcSwitch.add(src)
+            antiflap.portDown(src.switchId, src.portNo)
+        }
+
+        List<PathNode> broughtDownPortsIntermSwitch = []
+        altPaths.findAll { it.first().portNo == selectedPath.first().portNo &&
+                it[2].portNo != selectedPath[2].portNo && it[2].switchId == selectedPath[2].switchId
+        }.unique { it[2] }.each { path ->
+            def src = path[2]
+            broughtDownPortsIntermSwitch.add(src)
+            antiflap.portDown(src.switchId, src.portNo)
+        }
+
+        Wrappers.wait(antiflapMin + WAIT_OFFSET) {
+            assert northbound.getAllLinks().findAll {
+                it.state == FAILED
+            }.size() == (broughtDownPortsSrcSwitch.size() + broughtDownPortsIntermSwitch.size()) * 2
+        }
+
+        and: "Make reverse path not preferable. Forward path is still preferable(on the intermediate switch)"
```

Comment:
? Unrelated description

Reply:
why do you think it is unrelated?
I just tried to show that reverse path was updated

Reply:
'preferable' is used across the framework to describe paths with different costs (path with smaller cost is 'more preferable' than path with bigger cost) . You are manipulating bandwidth here and current description is not reflecting what is happening in test

Reply:
changed to `and: "Update reverse path to have not enough bandwidth to handle the flow"`